features/ubuntu_pro.feature

```diff
@@ -174,22 +174,84 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
 
     Examples: ubuntu release
       | release | machine_type |
-      | xenial  | aws.pro      |
       | xenial  | azure.pro    |
       | xenial  | gcp.pro      |
-      | bionic  | aws.pro      |
       | bionic  | azure.pro    |
       | bionic  | gcp.pro      |
-      | focal   | aws.pro      |
       | focal   | azure.pro    |
       | focal   | gcp.pro      |
-      | jammy   | aws.pro      |
       | jammy   | azure.pro    |
       | jammy   | gcp.pro      |
-      | noble   | aws.pro      |
       | noble   | azure.pro    |
       | noble   | gcp.pro      |
 
+  Scenario Outline: Auto-attach service works on Pro Machine on aws.pro
+    Given a `<release>` aws.pro machine with ubuntu-advantage-tools installed
+    When I run `systemctl start ua-auto-attach.service` with sudo
+    And I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+      """
+      contract_url: 'https://contracts.canonical.com'
+      data_dir: /var/lib/ubuntu-advantage
+      log_level: debug
+      log_file: /var/log/ubuntu-advantage.log
+      """
+    And I reboot the machine
+    And I run `pro status --wait` with sudo
+    And I run `pro security-status --format json` with sudo
+    Then stdout matches regexp:
+      """
+      "attached": true
+      """
+    When I run `journalctl --no-pager -b -u ua-auto-attach.service` with sudo
+    Then stdout matches regexp:
+      """
+      Auto-attaching: product code found on AWS
+      """
+    Then stdout does not match regexp:
+      """
+      Skipping auto-attach. Reason: No billingProduct nor marketplaceProductCode on AWS."
+      """
+
+    Examples: ubuntu release
+      | release |
+      | xenial  |
+      | bionic  |
+      | focal   |
+      | jammy   |
+      | noble   |
+
+  Scenario Outline: Auto-attach service works on Pro Machine on aws.generic
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I install ubuntu-advantage-pro
```

Comment:
@aciba90 let's instead install the `ubuntu-pro-auto-attach` package. The `ubuntu-advantage-pro` package is now just a transition package to this new one

Reply:
Done, thanks for the suggestion!